lib/collection/src/shards/local_shard.rs

```diff
@@ -492,11 +501,29 @@ impl LocalShard {
                 Err(err) => log::error!("{err}"),
                 Ok(_) => (),
             }
-            bar.inc(1);
+
+            // Update progress bar or show text progress every 100 operations
+            let progress = i as u64 + 1;
+            bar.set_position(progress);
+            if !show_progress_bar && progress % 100 == 0 {
```

Comment:
Should we maybe do something like `progress % (wal.len() / 10) == 0`? So that we always show 10 notifications at 10% progress each?

Reply:
Having a fixed number of output sounds like a great idea to me :+1:

Reply:
>  One idea would be to do a modulo 1000 to cut the noise.

> Should we maybe do something like progress % (wal.len() / 10) == 0?

Can do, but that could still take very long.

For example, in cloud we had 8500 entries two days ago, which took 5 hours to fully recover.

If we'd do 1000 or every 10% it would take half an hour for each progression update. I think that's too long as it'll look like it's stuck.

What about rendering an update every 1 or 5 minutes instead?

Reply:
Or, another idea: what if we just print progress every 10 seconds?

Reply:
> For example, in cloud we had 8500 entries two days ago, which took 5 hours to fully recover.

I would not expect such a low number of entries to take so long to recover but I guess this is a different issue.

> What about rendering an update every 1 or 5 minutes instead?

Every minute sounds good to me.

Reply:
> I would not expect such a low number of entries to take so long to recover but I guess this is a different issue.

In this specific case 100% of the entries were `DeletePointsByFilter` which can be extremely slow. These are exactly the ones for which rendering the progression is interesting.

> Every minute sounds good to me.

:+1:  I'll update the PR accordingly.

Reply:
@agourlay @ffuugoo Thanks for your input! It now renders progression every minute.